qemu/tests/uefishell.py

```diff
@@ -103,13 +103,17 @@ def var_copy(self):
         since the operations inside uefishell will be recorded
         into VAR file and may affact later test cases.
         """
-        source_var_path = "/usr/share/OVMF/OVMF_VARS.fd"
-        path = storage.get_image_filename_filesytem(
-            self.params, data_dir.get_data_dir())
-        vars_path = "%s.fd" % path
+        var_path = self.params["ovmf_path"] + '/'
+        var_name = self.params["ovmf_vars_filename"]
+        source_var_path = os.path.join(var_path + var_name)
```

Comment:
Please refer to the correct usage of `os.path.join`

Reply:
Ok,thanks a lot!